fastlane/Fastfile

```diff
@@ -18,7 +18,8 @@ files_with_version_number = {
   './Sources/Misc/SystemInfo.swift' => ['return "{x}"'],
   './.version' => ['{x}'],
   './scripts/docs/index.html' => ['purchases-ios-docs/{x}/documentation/revenuecat'],
-  './scripts/docs/v4_api_migration_guide.html' => ['purchases-ios-docs/{x}/documentation/revenuecat/v4_api_migration_guide']
+  './scripts/docs/v4_api_migration_guide.html' => ['purchases-ios-docs/{x}/documentation/revenuecat/v4_api_migration_guide'],
```

Comment:
I think we should remove this one?

Reply:
Not sure what you mean here. Remove the v4 migration guide?

Reply:
Not the file, but this line in `Fastfile`. We don't need to change the version there after 5.x right?

Reply:
I’m adding the v5 docs, and keeping the v4 docs, the only change in that line is adding a trailing comma. Or am I missing something?